tasks/request_from_archive/API.md

```diff
@@ -174,42 +175,39 @@ Perform a head request to get information about a file in S3.
 **Arguments**:
 
 - `s3_cli` - An instance of boto3 s3 client
-- `glacier_bucket` - The S3 bucket name
-- `file_key` - The key of the Glacier object
+- `archive_bucket_name` - The S3 bucket name
+- `file_key` - The key of the archived object
 
 **Returns**:
 
   None if the object does not exist.
   Otherwise, the dictionary specified in
-  https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/s3.html#S3.Client.head_object
+  https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/s3.html#S3
+  .Client.head_object
 
 <a id="request_from_archive.restore_object"></a>
 
 #### restore\_object
 
 ```python
-def restore_object(s3_cli: BaseClient,
-                   key: str,
-                   days: int,
-                   db_glacier_bucket_key: str,
-                   attempt: int,
-                   job_id: str,
-                   recovery_type: str = "Standard") -> None
+def restore_object(s3_cli: BaseClient, key: str, days: int,
+                   db_archive_bucket_key: str, attempt: int, job_id: str,
+                   recovery_type: str) -> None
```

Comment:
Since this is always called with the parameter present, the default was never used.